### PR TITLE
Add questions to git-quiz.ts (git cherry-pick and git tag)

### PR DESCRIPTION
**Pull request by Akshu-on-github**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: src/data/git-quiz.ts [hunk 1]
```diff
@@ -398,6 +398,28 @@ const gitQuiz = [
       "The git branch -r command shows the name of all remote branches that Git is tracking for the current repository.",
     Link: "https://www.freecodecamp.org/news/git-cheat-sheet/",
   },
+  {
+    Question:
+      "How can you apply the changes that a specific commit introduces",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:403]:
```suggestion
      "In Git,  which command is used to choose a commit from one branch and apply it to another?",
```

File: src/data/git-quiz.ts [hunk 2]
```diff
@@ -398,6 +398,28 @@ const gitQuiz = [
...
+    Answer: "git cherry-pick <commit>",
+    Distractor1: "git write-over <commit>",
+    Distractor2: "git goto <commit>",
+    Distractor3: "git merge <commit>",
+    Explanation:
+      "git-cherry-pick selects a handpicked commit and adds it to the working branch",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:409]:
```suggestion
      "git cherry-pick selects a handpicked commit from one branch and applies it to another branch.",
```

File: src/data/git-quiz.ts [hunk 3]
```diff
@@ -398,6 +398,28 @@ const gitQuiz = [
...
+    Link: "https://git-scm.com/docs/git-cherry-pick",
+  },
+  {
+    Question:
+      "How can you apply the changes that a specific commit introduces",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:414]:
```suggestion
      "In Git, what is the correct syntax for the git tag command?",
```

File: src/data/git-quiz.ts [hunk 4]
```diff
@@ -398,6 +398,28 @@ const gitQuiz = [
...
+    Answer: "git tag <tag-name>",
+    Distractor1: "git add-tag <tag-name>",
+    Distractor2: "git tag-add <tag-name>",
+    Distractor3: "git tag-on <commit>",
+    Explanation:
+      "git tag <tag_name> creates a tag object signed with GPG",
+    Link: "https://git-scm.com/docs/git-tag",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:421]:
```suggestion
    Link: "https://www.freecodecamp.org/news/git-tag-explained-how-to-add-remove/",
```

File: src/data/git-quiz.ts [hunk 5]
```diff
@@ -398,6 +398,28 @@ const gitQuiz = [
...
+  {
+    Question:
+      "How can you apply the changes that a specific commit introduces",
+    Answer: "git cherry-pick <commit>",
+    Distractor1: "git write-over <commit>",
+    Distractor2: "git goto <commit>",
+    Distractor3: "git merge <commit>",
+    Explanation:
+      "git-cherry-pick selects a handpicked commit and adds it to the working branch",
+    Link: "https://git-scm.com/docs/git-cherry-pick",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:410]:
```suggestion
    Link: "https://www.freecodecamp.org/news/the-git-cherry-pick-command/",
```

File: src/data/git-quiz.ts [hunk 6]
```diff
@@ -398,6 +398,28 @@ const gitQuiz = [
...
+  },
+  {
+    Question:
+      "How can you apply the changes that a specific commit introduces",
+    Answer: "git tag <tag-name>",
+    Distractor1: "git add-tag <tag-name>",
+    Distractor2: "git tag-add <tag-name>",
+    Distractor3: "git tag-on <commit>",
+    Explanation:
+      "git tag <tag_name> creates a tag object signed with GPG",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:420]:
```suggestion
      "git tag points to a specific point in the Git history and are often used to mark specific version releases in the code.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Akshu-on-github** [comment]
Hey there @jdwilkin4, I've made the requested changes, please review it once more

File: src/data/git-quiz.ts (hunk 1 above)
Comment (jdwilkin4) [reply src/data/git-quiz.ts:403]: You missed this review comment

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

**Comment by Akshu-on-github** [comment]
Ah my, bad - it should be ready now @jdwilkin4!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**